app/src/main/res/values-de/strings.xml

```diff
@@ -478,6 +478,8 @@ Signatur: %5$s</string>
     <string name="card_back_text">Hinterseite</string>
     <string name="title_activity_cards_detail">Karte hinzufügen</string>
     <string name="card_question">Frage</string>
+    <string name="moviesdetail_dots">Filmdetail</string>
+    <string name="ticketnumber_dots">Ticketnummer</string>
```

Comment:
We should provide a German translation for "Your Ticket". I would also prefer if the strings for an activity are marked by a comment before the corresponding part, e.g. 

\<!-- QR Code Ticket Activity -->
    \<string name="moviesdetail_dots">Filmdetails</string>
    \<string name="ticketnumber_dots">Ticketnummer</string>
    \<string name="your_ticket">Dein Ticket</string>

The same for the English strings.

Reply:
Also, I think section-titles like "Movie Details" or "Ticket Number" should have all nouns uppercase.

Reply:
Thank you. All fixed.